tests/textlab_api/textlab_api_test.py

```diff
@@ -46,7 +45,118 @@ def update_page_content(
     if hix_ignore:
         edit_page_data["hix_ignore"] = "on"
 
-    return (edit_page, admin_client.post(edit_page, edit_page_data))
+    return edit_page, admin_client.post(edit_page, edit_page_data)
+
+
+def create_page_content(
+    admin_client: Client,
+    title: str,
+    content: str,
+    hix_ignore: bool = False,
+) -> tuple[int, HttpResponse]:
+
+    new_page = reverse(
+        "new_page",
+        kwargs={
+            "region_slug": "augsburg",
+            "language_slug": "de",
+        },
+    )
+    new_page_data = {
+        "title": title,
+        "content": content,
+        "mirrored_page_region": "",
+        "_ref_node_id": 4,
+        "_position": "right",
+        "status": status.PUBLIC,
+    }
+    if hix_ignore:
+        new_page_data["hix_ignore"] = "on"
+
+    response = admin_client.post(new_page, new_page_data)
+    created_page_id_str = "0"
+    if search_result := re.search(r"\d+", response.headers.get("Location")):
+        created_page_id_str = search_result.group()
+    return int(created_page_id_str), response
+
+
+@pytest.mark.django_db
+def test_hix_score_create(
+    load_test_data: None,
+    admin_client: Client,
+    settings: SettingsWrapper,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check that the HIX score is requested and saved when the page translation is created
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param admin_client: The fixture providing the http client
+    :param settings: The fixture providing the django settings
+    :param mock_server: The fixture providing the dummy http server
+    """
+    # Setup a mocked Textlab API server with dummy responses
+    mock_server.configure("/user/login", 200, {"token": "dummy"})
+    mock_server.configure("/benchmark/420", 200, {"formulaHix": 15.12345678})
+
+    # Redirect call aimed at the Textlab API to the fake server
+    settings.TEXTLAB_API_URL = f"http://localhost:{mock_server.port}"
+
+    # Enable Textlab in the test region
+    Region.objects.filter(slug="augsburg").update(hix_enabled=True)
+
+    created_page_id, response = create_page_content(
+        admin_client,
+        "Test title: test_hix_score_create",
+        "test_hix_score_create: Neuer Inhalt",
+    )
+
+    assert response.status_code == 302
+
+    # Check the updated HIX score
+    page_translation = Page.objects.get(id=created_page_id).get_translation("de")
```

Comment:
```suggestion
    page_translation = Page.objects.latest("created_date").get_translation("de")
```
Then we don't need to extract the page_id from the response.